src/auth/src/credentials/mds.rs

```diff
@@ -19,7 +19,9 @@
 //! This is a local service to the VM (or pod) which (as the name implies) provides
 //! metadata information about the VM. The service also provides access
 //! tokens associated with the [default service account] for the corresponding
-//! VM.
+//! VM. The default host name of the metadata service is `metadata.google.internal`.
+//! If you would like to use a different hostname, you can set it using the
+//! `GCE_METADATA_HOST` environment variable.
```

Comment:
I think this deserves a new paragraph: it introduces a new concept or idea:

```suggestion
//! VM.
//!
//! The default host name of the metadata service is `metadata.google.internal`.
//! If you would like to use a different hostname, you can set it using the
//! `GCE_METADATA_HOST` environment variable.
```